src/appliance-onboarding-script/appliance_setup/run.py

```diff
@@ -103,6 +103,21 @@ def _populate_default_values_of_optional_fields_in_config(config):
     except IndexError:
         log_level = "INFO"
 
+    try:
+        isAutomated = (sys.argv[4].lower() == 'true') # isAutomated Parameter is set to true if it is an automation testing Run. 
```

Comment:
Instead of this manual parameter parsing, you can use built-in library ```argeparse```
https://docs.python.org/3/library/argparse.html

Reply:
Thanks for the suggestion, we will create a separate task for enhancements and address it there.